public/js/app.js

```diff
@@ -1,10 +1,24 @@
 // app.js
-$(document).on('click', '.load-more-comment', function (event) {
+document.addEventListener('click', function (event) {
+    if (!event.target.matches('.load-more-comment')) return;
     event.preventDefault();
-    $.get($(this).attr('href'), function (data) {
-        $('.load-more-comment-container').hide();
-        $('.comment-feed-container').append(data);
-    });
+    event.target.disabled = true;
+
+    var xhr = new XMLHttpRequest();
+    xhr.open('GET', event.target.href);
+    xhr.setRequestHeader('X-Requested-With', 'XMLHttpRequest');
+    xhr.send();
+    xhr.onreadystatechange = function() {
+        if (xhr.readyState === 4) {
+            if(xhr.status === 200) {
+                event.target.parentNode.parentNode.remove();
+                document.querySelector(".comment-feed-container").insertAdjacentHTML('beforeend', xhr.responseText);
```

Comment:
```suggestion
                document.querySelector(".comment-feed-container").insertAdjacentText('beforeend', xhr.responseText);
```

Reply:
What would be the benefit of treating the HTML we retrieved as text?

Reply:
Less possibiltiy of having unexpected XSS>

Reply:
Ah, in this case there's no sense in replacement. Good as is.